### ISSUE TITLE
Ansys license checkout failing

### ISSUE THREAD

**Issue by Prat9911** [ansys/pydpf-core#2605]
### Before submitting the issue

- [x] I have checked for [Compatibility](https://dpf.docs.pyansys.com/version/stable/getting_started/compatibility.html#id1) issues
- [x] I have searched among the existing issues
- [x] I am using a Python virtual environment

### Description of the bug

Ansys license checkout failed while trying to use Ansys DPF. Environment variable and Server address are set clearly.  
Received following error:- ansys.dpf.gate.errors.DPFServerException: license_checkout:0<-License increment: cfd_base. License check out failed. Ensure the server is set using environment variable ANSYSLMD_LICENSE_FILE to point to the server (Example ANSYSLMD_LICENSE_FILE = 1055@my_license_server.ansys.com). Ensure that you have valid license increments to use DPF Server. 

### Steps To Reproduce

1) Ran the following script in a command prompt:
from ansys.dpf import core as dpf
server = dpf.start_local_server(ansys_path=r"C:\Program Files\ANSYS Inc\v252")
server.apply_context(dpf.AvailableServerContexts.premium)
lic = dpf.LicenseContextManager(increment_name="cfd_base")

2) Received this error:
  raise errors.DPFServerException(sError.value)
ansys.dpf.gate.errors.DPFServerException: license_checkout:0<-License increment: cfd_base. License check out failed. Ensure the server is set using environment variable ANSYSLMD_LICENSE_FILE to point to the server (Example ANSYSLMD_LICENSE_FILE = 1055@my_license_server.ansys.com). Ensure that you have valid license increments to use DPF Server. For more information about accepted license increments, see https://dpf.docs.pyansys.com/version/stable/getting_started/licensing.html .

### Which Operating System causes the issue?

Windows

### Which DPF/Ansys version are you using?

Ansys 2025 R2

### Which Python version causes the issue?

3.13

### Installed packages

absl-py==2.3.1
ansys-dpf-core==0.14.1
ansys-dpf-post==0.10.1
antlr4-python3-runtime==4.9.3
anyio==4.10.0
argon2-cffi==25.1.0
argon2-cffi-bindings==25.1.0
arrow==1.3.0
asttokens==3.0.0
astunparse==1.6.3
async-lru==2.0.5
attrs==25.3.0
Automat==25.4.16
babel==2.17.0
beautifulsoup4==4.13.5
bleach==6.2.0
certifi==2025.8.3
cffi==1.17.1
charset-normalizer==3.4.3
colorama==0.4.6
comm==0.2.3
constantly==23.10.4
contourpy==1.3.3
cryptography==45.0.7
cssselect==1.3.0
cycler==0.12.1
debugpy==1.8.16
decorator==5.2.1
defusedxml==0.7.1
executing==2.2.1
fastjsonschema==2.21.2
filelock==3.19.1
flatbuffers==25.2.10
fonttools==4.59.2
fqdn==1.5.1
gast==0.6.0
google-pasta==0.2.0
grpcio==1.74.0
h11==0.16.0
h5py==3.14.0
httpcore==1.0.9
httpx==0.28.1
hydra-core==1.3.2
hyperlink==21.0.0
idna==3.10
imageio==2.37.0
imageio-ffmpeg==0.6.0
importlib_metadata==8.7.0
incremental==24.7.2
ipykernel==6.30.1
ipython==9.5.0
ipython_pygments_lexers==1.1.1
isoduration==20.11.0
itemadapter==0.12.2
itemloaders==1.3.2
jedi==0.19.2
Jinja2==3.1.6
jmespath==1.0.1
joblib==1.5.2
json5==0.12.1
jsonpointer==3.0.0
jsonschema==4.25.1
jsonschema-specifications==2025.9.1
jupyter-events==0.12.0
jupyter-lsp==2.3.0
jupyter_client==8.6.3
jupyter_core==5.8.1
jupyter_server==2.17.0
jupyter_server_terminals==0.5.3
jupyterlab==4.4.7
jupyterlab_pygments==0.3.0
jupyterlab_server==2.27.3
keras==3.11.3
kiwisolver==1.4.9
lark==1.2.2
libclang==18.1.1
lxml==6.0.1
Markdown==3.9
markdown-it-py==4.0.0
MarkupSafe==3.0.2
matplotlib==3.10.6
matplotlib-inline==0.1.7
mdurl==0.1.2
mistune==3.1.4
ml_dtypes==0.5.3
namex==0.1.0
narwhals==2.3.0
nbclient==0.10.2
nbconvert==7.16.6
nbformat==5.10.4
nest-asyncio==1.6.0
notebook_shim==0.2.4
numpy==2.3.2
omegaconf==2.3.0
opt_einsum==3.4.0
optree==0.17.0
packaging==25.0
pandas==2.3.2
pandocfilters==1.5.1
parsel==1.10.0
parso==0.8.5
pillow==11.3.0
platformdirs==4.4.0
plotly==6.3.0
pooch==1.8.2
prometheus_client==0.22.1
prompt_toolkit==3.0.52
Protego==0.5.0
protobuf==6.32.0
psutil==7.0.0
pure_eval==0.2.3
pyasn1==0.6.1
pyasn1_modules==0.4.2
pycparser==2.22
PyDispatcher==2.0.7
Pygments==2.19.2
pyOpenSSL==25.1.0
pyparsing==3.2.3
python-dateutil==2.9.0.post0
python-json-logger==3.3.0
pytz==2025.2
pyvista==0.46.3
pywin32==311
pywinpty==3.0.0
PyYAML==6.0.2
pyzmq==27.0.2
queuelib==1.8.0
referencing==0.36.2
requests==2.32.5
requests-file==2.1.0
rfc3339-validator==0.1.4
rfc3986-validator==0.1.1
rfc3987-syntax==1.1.0
rich==14.1.0
rpds-py==0.27.1
scikit-learn==1.7.1
scipy==1.16.1
scooby==0.10.1
Scrapy==2.13.3
seaborn==0.13.2
Send2Trash==1.8.3
service-identity==24.2.0
setuptools==80.9.0
six==1.17.0
sniffio==1.3.1
soupsieve==2.8
stack-data==0.6.3
tensorboard==2.20.0
tensorboard-data-server==0.7.2
tensorflow==2.20.0
termcolor==3.1.0
terminado==0.18.1
threadpoolctl==3.6.0
tinycss2==1.4.0
tldextract==5.3.0
tornado==6.5.2
tqdm==4.67.1
traitlets==5.14.3
Twisted==25.5.0
types-python-dateutil==2.9.0.20250822
typing_extensions==4.15.0
tzdata==2025.2
uri-template==1.3.0
urllib3==2.5.0
vtk==9.5.1
w3lib==2.3.1
wcwidth==0.2.13
webcolors==24.11.1
webencodings==0.5.1
websocket-client==1.8.0
Werkzeug==3.1.3
wheel==0.45.1
wrapt==1.17.3
yml==0.0.1
zipp==3.23.0
zope.interface==7.2

**Comment by PProfizi** [comment]
Hello @Prat9911,

I could not reproduce the issue on my side.
I think it may be related to the script being unable to reach the license server.

I have a few questions to help figure out what could be going wrong:
- is this something that used to work but does not anymore? If so, what was changed?
- are you able to run this script with other license increment names?
- are you able to run other PyDPF scripts?
- could you try running the following script to check the license server is reachable?

```py
from ansys.dpf import core as dpf

import os
import platform
import subprocess

os.environ["ANSYSLMD_LICENSE_FILE"] = "1055@my_license_server.ansys.com"
host = os.environ["ANSYSLMD_LICENSE_FILE"].split("@")[1]
param = '-n' if platform.system().lower()=='windows' else '-c'
print(subprocess.check_output(["ping", param, '1', host]).decode(encoding="utf-8"))

server = dpf.start_local_server(ansys_path=r"C:\Program Files\ANSYS Inc\v252")
server.apply_context(dpf.AvailableServerContexts.premium)
lic = dpf.LicenseContextManager(increment_name="cfd_base")
```

**Comment by Prat9911** [comment]
Hi @PProfizi ,
               Here are answers to your questions:
1) is this something that used to work but does not anymore? If so, what was changed?: This issue has been there since we started working with PdDPF. It never worked. Softwares from Ansys like fluent and others are opening without any license issue.
2) are you able to run this script with other license increment names?: We tried with CFD base, Prepost and others, same error came.
3) are you able to run other PyDPF scripts?: Just tried to run a file conversion script, from .cas to .vtu format. Same license error occured.
4) could you try running the following script to check the license server is reachable?: I tried running your script with different increments like cfd_base, prepost, but the same license error came.

Thanks.

**Comment by Prat9911** [comment]
I received following message in console:
Pinging WIN00379PTCL01P.gad.schneider-electric.com [10.222.190.18] with 32 bytes of data:
Reply from 10.222.190.18: bytes=32 time=8ms TTL=120

Ping statistics for 10.222.190.18:
    Packets: Sent = 1, Received = 1, Lost = 0 (0% loss),
Approximate round trip times in milli-seconds:
    Minimum = 8ms, Maximum = 8ms, Average = 8ms

**Comment by PProfizi** [comment]
Hi @Prat9911,
I took the liberty of removing potentially confidential information from the previous ping output.
So the server is reachable, which means we will need to investigate further.
I will let you know on Monday the course of action we can take.
